CHANGES.rst

```diff
@@ -10,6 +10,30 @@ New Features
 - ``SpectralRegion`` objects can now be created from the ``QTable``
   object returned from the line finding rountines. [#759]
 
+- Include new 6dFGS loaders. [#734]
+
+- Include new OzDES loaders. [#764]
+
+- Include new GAMA survey loaders. [#765]
+
+- Include new GALAH loaders. [#766]
+
+- Include new WiggleZ loaders. [#767]
+
+- Include new 2dF/AAOmega loaders. [#768]
+
+- Add loader to handle IRAF MULTISPEC non-linear 2D WCS. [#708]
+
+- Add ability to extract minimum bounding regions of ``SpectralRegion`` objects. [#755]
+
+- Implement new moment analysis function for specutils objects. [#758]
+
+- Add new spectral slab extraction functionality. [#753]
+
+- Include new loaders for AAT and other Australian surveys. [#719]
+
+- Move from Travis to GitHub actions. [#736]
```

Comment:
I'd put this in "other changes" (or not in the changelog at all since it's user-facing)
```suggestion
```